src/bp/ui-admin/src/Pages/Workspace/Bots/EditStageModal.tsx

```diff
@@ -165,14 +166,14 @@ const EditStageModal: FC<Props> = props => {
           />
         </FormGroup>
         {!isLastPipeline && (
-          <FormGroup label="Action">
+          <FormGroup label={lang('editStage.action')}>
             <RadioGroup onChange={({ currentTarget: { value } }) => setAction(value)} selectedValue={action} inline>
-              <Radio label="Copy" value="promote_copy" />
-              <Radio label="Move" value="promote_move" />
+              <Radio label={lang('editStage.copy')} value="promote_copy" />
+              <Radio label={lang('editStage.move')} value="promote_move" />
```

Comment:
Single words should be outside a namespace, i'm pretty sure we'll use those elsewhere :)